guardian/editorial-viewer
# Hide article correctly in social-media mode

## What does this change?

A few CSS tweaks to correctly hide the article body in social-media mode. Hopefully this implementation avoids a dependency on future changes to markup.

Before/after:

<img width="1674" alt="Screenshot 2021-02-10 at 17 01 55" src="https://user-images.githubusercontent.com/7767575/107544060-b9808580-6bc1-11eb-9a5c-1ab0760aab8a.png">

## How to test

Run this branch locally or in CODE, and click on 'Social share'. The results should look like the RHS of the screenshot.

## Dev notes

Oddly, `node-sass` wouldn't build on my machine, nor appear in CI until I bumped it to latest, so that bump's included in this patch, too. Hope that's OK.
